src/pages/waiting-room-to-car/waiting-room-to-car.html

```diff
@@ -9,106 +9,200 @@
-          </ion-col>
-        </ion-row>
+        </ion-col>
+        <ion-col>
+          <ion-row class="spacing-row"></ion-row>
+          <ion-row class="component-row" align-items-centre justify-content-start >
+              <ion-col col-32>
+                  <input type="radio" formControlName="eyesightCtrl" name="eyesightCtrl" id="eyesight-pass"
+                  class="gds-radio-button" [class.ng-invalid]="isCtrlDirtyAndInvalid('eyesightCtrl')"
+                  (click)="eyesightPassPressed()" [checked]="pageState.eyesightPassRadioChecked$ | async" value="P">
+                <label for="eyesight-pass" class="radio-label">Pass</label>
+              </ion-col>
+              <ion-col col-32>
+                  <input type="radio" formControlName="eyesightCtrl" name="eyesightCtrl" id="eyesight-fail"
+                  class="gds-radio-button" [class.ng-invalid]="isCtrlDirtyAndInvalid('eyesightCtrl')"
+                  (click)="eyesightFailPressed()" [checked] = "pageState.eyesightFailRadioChecked$ | async" value="F">
+                <label for="eyesight-fail" class="radio-label">Fail</label>
+              </ion-col>
+              <ion-col></ion-col>
+          </ion-row>
+          <ion-row class="validation-message-row" align-Items-centre justify-content-start>
+                  <div id="waiting-room-insurance-validation-text" class="validation-text"
+                  [class.ng-invalid]="isCtrlDirtyAndInvalid('eyesightCtrl')">
+                  Select the eyesight test outcome
+                  </div>
+          </ion-row>
+        </ion-col>
+      </ion-row>
         <ion-row *ngIf="pageState.eyesightFailRadioChecked$ | async">
           <eyesight-failure-confirmation [cancelFn]="eyesightFailCancelled"></eyesight-failure-confirmation>
         </ion-row>
-        <ion-row [hidden]="pageState.eyesightFailRadioChecked$ | async" id="post-eyesight-form-content">
-          <ion-col>
-            <ion-row class="mes-full-width-card" id="tell-me-card" align-items-center>
-              <ion-col col-32>
+        <ion-row no-padding [hidden]="pageState.eyesightFailRadioChecked$ | async" id="post-eyesight-form-content">
+          <ion-col class="no-padded-col">
+            <ion-row class="mes-validated-row" id="tell-me-card" align-items-center justify-content-start>
+              <div class="row-validation-bar"></div>
+              <ion-col class="component-label" col-32>
                 <label>Tell me question</label>
               </ion-col>
-            </ion-row>
-            <ion-row class="mes-full-width-card" id="registration-card" align-items-center>
-              <ion-col col-32>
-                <label>Registration number</label>
+              <ion-col>
+                  <ion-row class="spacing-row"></ion-row>
+                  <ion-row class="component-row" align-items-centre justify-content-start >
+                      <ion-col></ion-col>
```

Comment:
Is there a reason for loads of empty ion-col tags

Reply:
removed